x-pack/plugin/core/src/yamlRestTest/resources/rest-api-spec/test/license/10_feature_usage.yml

```diff
@@ -3,4 +3,4 @@
   - do:
       license.get_feature_usage: {}
 
-  - length: { features: 0 }
+  - length: { features: 1 }
```

Comment:
Failing due to the `.security` index now triggering a `write-load-forecast` feature:

```json
{	
  "stash" : {	
    "body" : {	
      "features" : [	
        {	
          "family" : null,	
          "name" : "write-load-forecast",	
          "context" : null,	
          "last_used" : "2025-01-20T14:46:31.304Z",	
          "license_level" : "enterprise"	
        }	
      ]	
    }	
  }	
}
```